### ISSUE TITLE
Relaunch blocking tasks

### ISSUE THREAD

**Issue by musjj** [Canop/bacon#216]
It looks like that blocking tasks (e.g. running a web server) aren't automatically restarted when there are changes. Is there a way to configure so that it's automatically killed and restarted?

**Comment by musjj** [comment]
Oh it looks like that there was a change in the last version:

https://github.com/Canop/bacon/releases/tag/v2.21.0

> With `on_change_strategy = "kill_then_restart"`, the current job is immediately killed and a new job restarted. This is the behavior that bacon had before this PR. It has the downside of never allowing any job to complete if you're always changing files and the job is just a little too long to finish between changes.

**Comment by Canop** [comment]
Yes. I reopen the issue because I should improve the default bacon.toml file and maybe the doc to explain that

**Comment by Canop** [comment]
website updated: https://dystroy.org/bacon/config/#jobs

But I'll add a comment in the default bacon.toml too
